extensions/amp-story-shopping/0.1/amp-story-shopping-tag.css

```diff
@@ -13,6 +13,14 @@ amp-story-shopping-tag {
   --i-amphtml-shopping-tag-backdrop-filter: blur(20px);
 }
 
+
+.amp-story-shopping-tag-currency-symbol {
+  font-family: 'Poppins', sans-serif !important;
```

Comment:
How is this font loaded? Should it have a fallback system font?

Reply:
The font is loaded in `amp-story-shopping-tag.js`, in the `buildCallback` see the `FONTS_TO_LOAD` variable and the `loadFonts_` function. I will consult with the UX Designer for the fallback font.